packages/@o3r/workspace/schematics/ng-add/project-setup.ts

```diff
@@ -67,6 +68,7 @@ export const prepareProject = (options: NgAddSchematicsSchema): Rule => {
       generateRenovateConfig(ownSchematicsFolder),
       addVsCodeRecommendations(vsCodeExtensions),
       updateGitIgnore(workspaceConfig),
+      addLerna(ownPackageJsonContent),
```

Comment:
Maybe you can add a config to select the Monorepo manager (and so probably retrain the choice to `lerna` today)